### ISSUE TITLE
Make accounts not hardcoded

### ISSUE THREAD

**Issue by Sharktroid** [rpekash/Challenge-1#7]
Accounts are currently hardcoded. They should instead be retrieved from a database, and it should be possible to create accounts as well.

**Comment by rpekash** [comment]
I agree with this, allowing the user base to grow passively.

**Comment by Tbryant7** [comment]
I agree with this aswell makes much more sense for users to be allowed to make their own accounts and passwords
